### DISCUSSION TITLE
decimal digits for the indicator values

### DISCUSSION THREAD

**Discussion by ukraine-sdg** [open-sdg/open-sdg#1469] [Q&A] [answered]
how can I control the number of decimal digits. In particular, how I can make 1.0 instead of 1?

**Comment by christoph-LSN** [comment, marked as answer]
@ukraine-sdg have a look here: [https://open-sdg.readthedocs.io/en/latest/indicator-configuration/#precision](https://open-sdg.readthedocs.io/en/latest/indicator-configuration/#precision)

**Reply by ukraine-sdg** [reply]
Thank you, Christoph!

Were should I write "precision: decimals: 2"? 
In which file?

**Reply by christoph-LSN** [reply]
I am not familiar with the indicator-config file, but take a look [here](https://ukraine-sdg.github.io/site/1-1-1/config/) Scrolling down to precision

**Reply by brockfanning** [reply]
@ukraine-sdg Before editing indicator configuration, there are a couple of site configurations that need to be corrected. Please [merge this PR](https://github.com/ukraine-sdg/site/pull/181) to start, then wait about 10 minutes for your site to rebuild, and finally continue below.

The basic steps for editing indicator configuration to set the precision to 1 decimal are these:

1. Go to the indicator on your site
2. Click on the "Edit" tab
3. Click on "Edit Configuration"
4. Scroll down to "Precision rules" and expand it
5. Click "+Precision rule" to add one
6. Leaving "Series" and "Unit" blank, enter "1" under "Decimals"
7. At the top, click "Download configuration" to save the indicator configuration file
8. Click on "Go to repository"
9. Upload the configuration file

**Comment by christoph-LSN** [comment]
@ukraine-sdg may this ist also a usefull information: [https://open-sdg.readthedocs.io/en/latest/configuration/#decimal_separator](https://open-sdg.readthedocs.io/en/latest/configuration/#decimal_separator)
